README.md

```diff
@@ -0,0 +1,18 @@
+# bit-docs
+
+Loosely based on ideas and lessons learned from [DocumentJS](http://documentjs.com), bit-docs is an evolving set of tools that allow you to:
+
+ - Write documentation inline or in markdown files.
+ - Specify your code's behavior precisely with JSDoc
+   and [Google Closure Compiler](https://developers.google.com/closure/compiler/docs/js-for-compiler)
```

Comment:
I think this is the new link: https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler